bodhi/client/__init__.py

```diff
@@ -215,6 +217,7 @@ def _save_override(url, user, password, staging, edit=False, **kwargs):
     resp = client.save_override(nvr=kwargs['nvr'],
                                 duration=kwargs['duration'],
                                 notes=kwargs['notes'],
+                                wait=kwargs['wait'],
```

Comment:
We don't want the server or the bindings to do the waiting, just the CLI.

If the server does the wait, that will hold the connection open for a long time (Koji regenerates repos every 15 minutes in Fedora's config, which means that this could cause a connection to stay open for up to 15 minutes.)